homework/index.js

```diff
@@ -29,23 +31,127 @@
     return elem;
   }
 
+  function addRow(table, title, repoText) {
+    const tr = createAndAppend('tr', table);
+    createAndAppend('th', tr, {
+      text: title,
+    });
+    createAndAppend('td', tr, { text: repoText });
+    return tr;
+  }
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const date = new Date(repo.updated_at);
+    const li = createAndAppend('li', ul, { class: 'li-root' });
+    const table = createAndAppend('table', li);
+    const firstRow = addRow(table, 'Repository:', '');
+    createAndAppend('a', firstRow.lastChild, {
+      href: repo.html_url,
+      text: repo.name,
+      target: '_blank',
+    });
+
+    addRow(table, 'Description:', repo.description, 'td');
+    addRow(table, 'Forks:', repo.forks, 'td');
+    addRow(table, 'Updated:', date.toLocaleDateString(), 'td');
+  }
+
+  function createOptions(repo, parent, index) {
+    createAndAppend('option', parent, {
+      text: repo.name,
+      value: index,
+    });
+  }
+  function selections(selection, data, parent) {
+    const ul = createAndAppend('ul', parent, {
+      class: 'ul-root',
+    });
+    const repo = data[selection];
+    renderRepoDetails(repo, ul);
+  }
+  function createContributorsSection(url, parent) {
+    fetchJSON(url).then(data => {
```

Comment:
Again, don't use the name `data` if you have more precise knowledge about what you are getting here: you are fetching an array of contributor objects, therefore the logical name would be `contributors`.